tests/CMakeLists.txt

```diff
@@ -31,11 +31,19 @@ add_custom_target(test_defrag
                   COMMAND "${CMAKE_CURRENT_BINARY_DIR}/eblob_defrag_test"
                   DEPENDS ${TESTS_DEPS} eblob_defrag_test)
 
+
+add_executable(eblob_binlog_test unit/binlog_test.cpp unit/eblob_wrapper.cpp)
```

Comment:
It should be in the next commit with tests

Reply:
fixed